.github/workflows/cloud-workflow.yml

```diff
@@ -0,0 +1,175 @@
+---
+name: cloud-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+      - 'feg/**'
+      - 'cwf/**'
+  pull_request:
+    branches:
+      - master
+jobs:
+  insync-checkin:
```

Comment:
Original jobs had comments about the purpose of the job. Let's preserve those over to this workflow as well

E.g. https://github.com/magma/magma/blob/5b3bc00420ce3a219eeda065b259aeb7fa7a222b/.circleci/config.yml#L624-L625